common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -258,20 +270,17 @@
             };
 
             DiscussionThreadListView.prototype.renderThreads = function() {
-                var content, rendered, thread, _i, _len, _ref;
-                this.$(".forum-nav-thread-list").html("");
-                rendered = $("<div></div>");
-                _ref = this.displayedCollection.models;
-                for (_i = 0, _len = _ref.length; _i < _len; _i++) {
-                    thread = _ref[_i];
-                    content = this.renderThread(thread);
-                    rendered.append(content);
+                var $content, thread, i, len;
+                this.$(".forum-nav-thread-list").empty();
+                for (i = 0, len = this.displayedCollection.models.length; i < len; i++) {
```

Comment:
Thanks for cleaning this up, the Coffeescript generated code here was a little gross.